### ISSUE TITLE
Can't save a BentoService with TorchScript model

### ISSUE THREAD

**Issue by priancho** [bentoml/BentoML#1388]
**Describe the bug**

Hi,

I have created a sample BentoService program that uses TorchScript model based on the [official example](https://docs.bentoml.org/en/latest/frameworks.html#pytorch).
But it fails with the following error (please see sample program I used and error messages in section 2 for details).

This problem seems like related to https://github.com/pytorch/pytorch/issues/18106.
However, PyTorch 1.6 release note mentions that it fixed copy.deepcopy() problem :-(

**To Reproduce**

I used the following code for test.

```python
import torch
import torch.nn as nn

import bentoml
from bentoml.adapters import ImageInput
from bentoml.frameworks.pytorch import PytorchModelArtifact


class Net(nn.Module):
    def __init__(self):
        super(Net, self).__init__()

    def forward(self, x):
        return x+1

net = Net()


@bentoml.env(infer_pip_packages=True)
@bentoml.artifacts([PytorchModelArtifact('net')])
class PytorchModelService(bentoml.BentoService):

    @bentoml.api(input=ImageInput(), batch=True)
    def predict(self, imgs):
        outputs = self.artifacts.net(imgs)
        return outputs

svc = PytorchModelService()

# Pack a TorchScript Model
traced_net = torch.jit.script(net)
svc.pack('net', traced_net)

bs_path = svc.save('test_bentoservice')
print(bs_path)
```

Running the program above fails with the following error messages.

```bash
Traceback (most recent call last):
  File "test.py", line 35, in <module>
    bs_path = svc.save('test_bentoservice')
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/service/__init__.py", line 669, in save
    return yc.repository.upload(self, version, labels)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/yatai/client/bento_repository_api.py", line 134, in upload
    save_to_dir(bento_service, tmpdir, version, silent=True)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/saved_bundle/bundler.py", line 223, in save_to_dir
    _write_bento_content_to_dir(bento_service, path)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/saved_bundle/bundler.py", line 86, in _write_bento_content_to_dir
    bento_service.artifacts.save(module_base_path)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/service/artifacts/__init__.py", line 258, in save
    artifact.save(save_path)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/service/artifacts/__init__.py", line 181, in wrapped_save
    return original(*args, **kwargs)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/frameworks/pytorch.py", line 116, in save
    return cloudpickle.dump(self._model, open(self._file_path(dst), "wb"))
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/utils/cloudpickle.py", line 1091, in dump
    CloudPickler(file, protocol=protocol).dump(obj)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/bentoml/utils/cloudpickle.py", line 367, in dump
    return Pickler.dump(self, obj)
  File "/home/hancheol/anaconda3/lib/python3.8/pickle.py", line 485, in dump
    self.save(obj)
  File "/home/hancheol/anaconda3/lib/python3.8/pickle.py", line 576, in save
    rv = reduce(self.proto)
  File "/home/hancheol/anaconda3/lib/python3.8/site-packages/torch/jit/_script.py", line 590, in __getstate__
    raise pickle.PickleError(
_pickle.PickleError: ScriptModules cannot be deepcopied using copy.deepcopy or saved using torch.save. Mixed serialization of script and non-script modules is not supported. For purely script modules use my_script_module.save(<filename>) instead.
```

**Expected behavior**

A BentoService should be built w.o error.

**Screenshots/Logs**

**Environment:**
 - OS: Ubuntu 18.04
 - Python Version: 3.8.1
 - BentoML Version: 0.10.1

**Additional context**

**Comment by withsmilo** [comment]
Hi, @priancho .
Some enhancement commits about Pytorch JIT traced/scripted models were merged with `master` branch, but not released yet. Please refer https://github.com/bentoml/BentoML/pull/1293. This will probably be included in the next release. Maybe next Monday? :blush:

**Comment by parano** [comment]
Thanks @withsmilo @priancho the support for Pytorch JIT traced model has been released in version 0.11.0 😊
